lib/fastimage.rb

```diff
@@ -585,7 +585,7 @@ def animated?
 
       # fields (1) + bg color (1) + pixel ratio (1)
       fields = @stream.read(3).unpack("CCC")[0]
-      if fields & 0x80 # Global Color Table
+      if fields & 0x80 != 0 # Global Color Table
```

Comment:
Or maybe
```
if fields[10].zero?
```

Reply:
I think that would have worked if it were `fields[7]`, but I do not think that makes the code more readable.